generator/internal/discovery_type_vertex.h

```diff
@@ -121,6 +124,7 @@ class DiscoveryTypeVertex {
   std::string name_;
   std::string package_name_;
   nlohmann::json json_;
+  google::protobuf::DescriptorPool const* const descriptor_pool_;
```

Comment:
Q.  Why make `descriptor_pool_` const when other members that similarly go unchanged are not const?